frontend/components/pod_components/TraineePod.js

```diff
@@ -2,52 +2,57 @@ import React from 'react';
 import { StyleSheet, Text, View, TouchableOpacity, SafeAreaView } from 'react-native';
 import Constants from 'expo-constants';
 import axios from 'axios';
+import { getAccessToken } from '../../utils/auth.js';
 
 import PodProgressBar from './PodProgressBar.js';
 
-const COMPET_TOTAL_OUTCOMES = 3
-const CAREER_TOTAL_OUTCOMES = 2
-const LIFE_TOTAL_OUTCOMES = 2
-
 const server_add = Constants.manifest.extra.apiUrl;
 
 export default class TraineePodScreen extends React.Component {
     /* State variables initalized:
      * 	  - compet_outcomes: number of completed competency outcomes 
      * 	  - career_outcomes: number of completed career pathway outcomes
      * 	  - life_outcomes: number of completed life essential outcomes 
+     *    - compet_total_outcomes: total number of competency outcomes
+     *    - career_total_outcomes: total number of career outcomes
+     *    - life_total_outcomes: total number of life outcomes
      */
     state = {
         compet_outcomes: 0, 
         career_outcomes: 0,
         life_outcomes: 0,
+        compet_total_outcomes: 0,
+        career_total_outcomes: 0,
+        life_total_outcomes: 0,
```

Comment:
I would try to think of a way to have as many of these as you fetch. For now this works as intended!